### DISCUSSION TITLE
verifying previously written data using a config file

### DISCUSSION THREAD

**Discussion by celesteking** [axboe/fio#1759] [Q&A]
I'd like to have 2 sections in my FIO config file. I'd like to run them sequentially, one by one.

1. First section would define a write job with one name
2. Second section would define the verification job with a different name

So far, I'm unable to come up with a solution as the first obstacle I've encountered is that FIO is trying to load a state file that doesn't exist.

**Comment by axboe** [comment]
See ```stonewall```. For specific issues, please be more detailed on what exactly is being run.

**Reply by celesteking** [reply]
```ini
[global]

iodepth=16
blocksize=4k

ioengine=libaio
direct=1

randrepeat=1
invalidate=1

verify=crc32c-intel
verify_backlog=1024
verify_dump=1
do_verify=0

numjobs=1

group_reporting
stonewall

[randwrite]
time_based
runtime=3

do_verify=0
verify_state_save=1

rw=randwrite

[verify]
verify_state_load=1

do_verify=1
verify_only=1

rw=randread
```
Results in 
```
Starting 2 processes
fio: open state file: No such file or directory
fio: state file: local-verify-1-verify.state (for_write=0)
```
and a failed verification run.
